touch_element/test_apps/README.md

```diff
@@ -0,0 +1,2 @@
+| Supported Targets | ESP32-S2 | ESP32-S3 |
```

Comment:
actually this table is not necessary, we're not in the esp-idf repo, there's no check here.